envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/endpoints/EnvoyEndpointsFactory.kt

```diff
@@ -37,9 +37,7 @@ class EnvoyEndpointsFactory(
                         val locality = it.locality
```

Comment:
can we change `mapNotNull` to `map` now? I think the code cannot return null now.

Reply:
true, changed :)